### PR TITLE
chore: Removing the old BINDING_SUCCESS event

### LINKED ISSUE #22893: [Task]: Stop tracking the BINDING_SUCCESS event
### Is there an existing issue for this?

- [X] I have searched the existing issues

### SubTasks

Now that we have the new `ENTITY_BINDING_SUCCESS` event firing correctly, let's stop tracking the `BINDING_SUCCESS` event on Mixpanel

### PR DESCRIPTION
**Pull request by Druthi**
## Description
Stop tracking the BINDING_SUCCESS event

Fixes #22893 

## Type of change
- Chore (housekeeping or task changes that don't impact user perception)

## How Has This Been Tested?
- Manual

### Dev activity
- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] PR is being merged under a feature flag

### QA activity:
- [ ] Test plan has been approved by relevant developers
- [ ] Test plan has been peer reviewed by QA
- [ ] Cypress test cases have been added and approved by either SDET or manual QA
- [ ] Organized project review call with relevant stakeholders after Round 1/2 of QA
- [ ] Added Test Plan Approved label after reveiwing all Cypress test

### CODE REVIEW

**Review by rimildeyjsr — APPROVED** [review]

File: app/client/src/sagas/PostEvaluationSagas.ts [hunk 1]
```diff
@@ -382,14 +382,6 @@ export function* logSuccessfulBindings(
...
-          // for first evaluation, bindings are not added by user hence skipping it.
-          AnalyticsUtil.logEvent("BINDING_SUCCESS", {
-            unevalValue,
-            entityType,
-            propertyPath,
-          });
-
           /**Log the binding only if it doesn't already exist */
           if (
             !successfulBindingPaths[evaluatedPath] ||
```
Comment (SatishGandham) [inline app/client/src/sagas/PostEvaluationSagas.ts:387]: Are we calling this new event whenever a bound path gets a new value?

Comment (Druthi) [reply app/client/src/sagas/PostEvaluationSagas.ts:387]: > Are we calling this new event whenever a bound path gets a new value?

New value as in? 

We call the event whenever there is a new BINDING.

**Pull request merged**